jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/diagram/ProcessInstanceDiagramViewImpl.html

```diff
@@ -1,7 +1,24 @@
 <!DOCTYPE html>
 <div class="row kie-process-instance-nodes" style="overflow:auto; padding-top: 20px;">
-    <div class="col-md-2" data-field="node-actions-panel">
-        <div class="panel panel-default">
+    <div class="col-md-2" data-field="diagram-details">
+        <div class="panel panel-default" data-field="parent-sub-process">
+            <div class="panel-heading">
+                <h3 class="panel-title" data-i18n-key="ParentProcessInstance"></h3>
+            </div>
+            <div class="panel-body">
+                <a class="btn-link form-control-static" data-field="parent-process-name" ></a>
```

Comment:
To keep consistency with the process breadcrumb, I suggest to instead of breaking line to show the ID, please use the format of Process Instance Id - Process Name. Ex: 2 - place-order